test/interpret.jl

```diff
@@ -942,7 +942,16 @@ end
 @test @interpret foo_536(UInt8('A'))
 
 @static if isdefined(Base.Experimental, Symbol("@opaque"))
-    @test @interpret (Base.Experimental.@opaque x->3*x)(4) == 12
+    @testset "opaque closures" begin
+        g(x) = 3x
+        f = Base.Experimental.@opaque x -> g(x)
+        @test @interpret f(4) == 12
```

Comment:
```suggestion
        @test (@interpret f(4)) == 12
```
Otherwise this interprets the call to `==`.

Reply:
Ah, good catch! I guess it doesn't really matter whether we also interpret the `==` here, but I can make a follow-up PR to make this more consistent.